OpenSource/OpenSource.mdw

```diff
@@ -0,0 +1,3 @@
+<WorkspaceItem ctype="Workspace">
```

Comment:
Remove this file too.

Reply:
okay

Reply:
@gauravano : i have updated the pull request . Thanks.